app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -87,12 +87,12 @@ def index
           repos = scoped_search(*base_args, options)
           csv_response(repos,
                        [:id, :name, :description, :label, :content_type, :arch, :url, :major, :minor,
-                        :cp_label, :content_label, :pulp_id, :container_repository_name,
+                        :content_label, :pulp_id, :container_repository_name,
                         :download_policy, 'relative_path', 'product.id', 'product.name',
                         'environment_id'],
                        ['Id', 'Name', 'label', 'Description', 'Content Type', 'Arch', 'Url', 'Major', 'Minor',
```

Comment:
Oh since you noticed others were out of whack, I double checked too and label and description are reversed here, too.

Reply:
Updated!